gluefactory/datasets/oxford_paris_mini_1view_jpldd.py

```diff
@@ -90,6 +109,38 @@ def _init(self, conf):
         }
         print(f"DATASET OVERALL(NO-SPLIT) IMAGES: {len(images)}")
 
+        augmentation_map = {
+           "dark": augmentations.DarkAugmentation,
+            "lg": augmentations.LGAugmentation,
+            "identity": augmentations.IdentityAugmentation
+        }
+
+        self.augmentation = augmentation_map[self.conf.load_features.augment.type]()
+
+    def download_binary_df_gt(self):
```

Comment:
I would prefer if we have everything in one download. It makes things easier. 
I can give you the directions to our current gt that contains everything so you can add the line gt to it and we can reupload. What do you think? (I would also be okay with keeping it that way if it works fine)